dagql/introspection/types.go

```diff
@@ -263,7 +263,7 @@ func Install[T dagql.Typed](srv *dagql.Server) {
 			if self.Value == nil {
 				return dagql.Null[dagql.String](), nil
 			} else {
-				return dagql.NonNull(dagql.NewString(self.Value.Raw)), nil
+				return dagql.NonNull(dagql.NewString(self.Value.String())), nil
```

Comment:
Maybe `.String()` here is adding the quotes...?

Reply:
(see above)